modules/views/components/civicrm.core.inc

```diff
@@ -157,6 +157,18 @@ function _civicrm_core_data(&$data, $enabled) {
     ),
   );
 
+  $data['civicrm_event']['civicrm_participant'] = array(
+    'real field' => 'id',
+    'title' => t('CiviCRM Participant (starting from Event)'),
+    'help' => t('Allows a Participant Contact relationship from an Event'),
+    'relationship' => array(
+      'base' => 'civicrm_participant',
+      'base field' => 'event_id',
+      'handler' => 'civicrm_handler_relationship_participant',
+      'label' => t('CiviCRM Participant (starting from Event)'),
```

Comment:
I don't understand this string (note that it appears twice, if we decide to change it).
What about "CiviCRM Event Participant"

Reply:
Hi @colemanw - this string appears twice in the same way that it does for CiviCRM Relationship. I also looked to "starting from Contact A" as guidance, although here I see it's probably not necessary. I've adjusted the PR accordingly.